packages/matter-node-ble.js/src/ble/NobleBleChannel.ts

```diff
@@ -79,6 +80,10 @@ export class NobleBleCentralInterface implements NetInterface {
                 `Peripheral ${address.peripheralAddress} is already connected. Only one connection supported right now.`,
             );
         }
+        if (peripheral.state !== "disconnected") {
+            // Try to cleanup strange in between states
```

Comment:
Strange = storage?

Reply:
no really 

strange "in between states"

Reply:
adjustes text